test/framefusion.test.ts

```diff
@@ -73,6 +73,19 @@ describe('FrameFusion', () => {
         await extractor.dispose();
     });
 
+    it.only('can get duration from webm', async() => {
+        // Arrange
+        const extractor = await BeamcoderExtractor.create({
+            inputFileOrUrl: 'https://storage.googleapis.com/lumen5-prod-video/anita-6uTzyZtNRKztypC.webm',
+        });
+
+        // Act and Assert
+        expect(extractor.duration).to.equal(115);
```

Comment:
@animanathome can we use test data which we inspect in this repo?

Reply:
I'm not sure what you mean with that @stepancar

Reply:
This doesn't work for you locally?

Reply:
@animanathome I mean, can we put that video to this repo directly so it would be easy to call ffprobe